lib/sourceror.ex

```diff
@@ -906,4 +912,16 @@ defmodule Sourceror do
     |> Enum.chunk_every(2)
     |> Enum.map(&Enum.join/1)
   end
+
+  defp locals_without_parens() do
+    if Version.match?(System.version(), ">= 1.13.0") do
+      File.cwd!()
+      |> Path.join(".formatter.exs")
+      |> Mix.Tasks.Format.formatter_for_file()
+      |> elem(1)
+      |> Keyword.get(:locals_without_parens, [])
```

Comment:
`formatter_for_file` does not need the `.formatter.exs` file. The function wants the file that needs to be formatted. However, in the end the extension of the given file name is essential. 
```suggestion
      {_formatter, formatter_opts} = Mix.Tasks.Format.formatter_for_file("elixir.ex")
      Keyword.get(formatter_opts, :locals_without_parens, [])
```